example_export_request.json

```diff
@@ -1,11 +1,11 @@
 {
     "name": "Example Export Request",
     "format": "json",
+    "expires": "2022-01-01T00:00:00Z",
```

Comment:
Does the service need to accept the `expires` date?  Or is this a field that should be filled in by the service?